vllm/config/parallel.py

```diff
@@ -170,6 +170,13 @@ class is dynamically inherited by the worker class. This is used to inject
     Set to be private as it's not intended to be configured by users.
     """
 
+    enable_context_parallel: bool = False
+    """Use for enable context parallel. Default is False."""
```

Comment:
We do want to support `context_parallel_size < tp_size` in the near future, let's remove this flag and only use `context_parallel_size`, while asserting `context_parallel_size == 1 or context_parallel_size == tp_size` now.

Reply:
done.